app/src/main/res/values/strings_backup.xml

```diff
@@ -22,26 +22,26 @@
     <string name="backup_desc_import_crypt">Restore accounts from a password-protected JSON file</string>
     <string name="backup_desc_import_openpgp">Restore accounts from an OpenPGP-encrypted JSON file</string>
 
-    <string name="backup_desc_crypt_setup">You need to set a <b>Backup password</b> in the
+    <string name="backup_desc_crypt_setup">Please, you need to set a <b>Backup password</b> in the
         <b>Settings</b> to use encrypted backups.
     </string>
 
-    <string name="backup_desc_openpgp_provider">You need to install an OpenPGP provider and enable
-        it in the <b>Settings</b> to use this feature.
+    <string name="backup_desc_openpgp_provider">Please, you need to install an OpenPGP provider and enable
+        it in the <b>Settings</b> page in order to use this feature.
     </string>
-    <string name="backup_desc_openpgp_keyid">You need to select an OpenPGP key in the <b>Settings</b>
-        before you can create encrypted backups.
+    <string name="backup_desc_openpgp_keyid">Please, you need to select an OpenPGP key in the <b>Settings</b>
+        page before you can create encrypted backups.
     </string>
```

Comment:
I'm not sure about the `Please, you` in the three strings above. In my opinion we should either keep it as only `You need to ...` or drop the `you need to` completely and only use the `Please` (e.g. `Please install an OpenPGP provider ...`). Maybe @RichyHBM can chime in here as well for a third opinion.

Reply:
Agree with @flocke

Reply:
Ok. Point noted. Would effect the chances ASAP